perllib/FixMyStreet/App/Model/PhotoSet.pm

```diff
@@ -117,6 +118,23 @@ has images => ( #  AoA of [$fileid, $binary_data] tuples
                     return ();
                 }
 
+                # base64 decode the file if it's encoded that way
+                # Catalyst::Request::Upload doesn't do this automatically
+                # unfortunately.
+                if ($upload->headers->header('Content-Transfer-Encoding') eq 'base64') {
```

Comment:
This gives an undefined warning if that header does not exist, perhaps stick it in a variable first and check it does exist too?